vespa_feed_perf/src/test/java/com/yahoo/vespa/feed/perf/FeederParamsTest.java

```diff
@@ -100,6 +100,32 @@ public void requireThatNumMessagesToSendAreParsed() throws ParseException, FileN
         assertEquals(19, new FeederParams().parseArgs("--nummessages", "19").getNumMessagesToSend());
     }
 
+    @Test
+    public void requireThatWindowSizeIncrementIsParsed() throws ParseException, FileNotFoundException {
+        assertEquals(20, new FeederParams().getWindowIncrementSize());
+        assertEquals(17, new FeederParams().parseArgs("--window_incrementsize", "17").getWindowIncrementSize());
+    }
+
+    static final double SMALL_NUMBER = 0.000000000001;
```

Comment:
"epsilon" might be a more familiar term for this when used with floating point comparisons

Reply:
Done